apps/hammond/docker-compose.yml

```diff
@@ -37,3 +41,9 @@ services:
       traefik.http.routers.hammond-local.entrypoints: websecure
       traefik.http.routers.hammond-local.service: hammond
       traefik.http.routers.hammond-local.tls: true
+    healthcheck:
+      test: wget --no-verbose --tries=1 --spider http://localhost:3000
```

Comment:
Shouldn't this be ?
```suggestion
      test: wget --no-verbose --tries=1 --spider http://localhost:${APP_PORT}
```

Reply:
Nop, it's internal port in docker container for app.

Reply:
Can be a bar only if you can change the binding port as I do for SFTPGo. With healthcheck it's better to do internal. For external port you can use something like uptime kuma instead

Reply:
You are correct actually!